FOS.Users.Api/FOS.Users.Api/FOS.Users.Api/Controllers/UserManagementController.cs

```diff
@@ -0,0 +1,177 @@
+﻿using AutoMapper;
+using FOS.Infrastructure.Commands;
+using FOS.Infrastructure.Queries;
+using FOS.Models.Entities;
+using FOS.Models.Requests;
+using FOS.Models.Responses;
+using MediatR;
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using static FOS.Models.Constants.Constants;
+
+namespace FOS.Users.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserManagementController : FOSControllerBase
+    {
+
+
+        private readonly IConfiguration _configuration;
```

Comment:
Hi @karthikrossi46 , Remove unused variables. Never inject IConfiguration. It contains huge data and might choke the Api.,